### PR TITLE
Added interactive Thanos e2e example.

### PR DESCRIPTION
**Pull request by bwplotka**
cc @matej-g 

Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: examples/interactive/interactive_test.go [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+)
+
+func exec(cmd string, args ...string) error {
+	if o, err := execlib.Command(cmd, args...).CombinedOutput(); err != nil {
+		return errors.Wrap(err, string(o))
+	}
+	return nil
+}
+
+func createData() (perr error) {
```
[the remark this answers is not in the document]
Comment (bwplotka) [reply examples/interactive/interactive_test.go:43]: what

File: examples/interactive/interactive_test.go [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+	maxTimeFresh = `2021-07-27T00:00:00Z`
+	maxTimeOld   = `2021-07-20T00:00:00Z`
+
+	store1Data = func() string { a, _ := filepath.Abs(filepath.Join(data, "store1")); return a }()
+	store2Data = func() string { a, _ := filepath.Abs(filepath.Join(data, "store2")); return a }()
+	prom1Data  = func() string { a, _ := filepath.Abs(filepath.Join(data, "prom1")); return a }()
+	prom2Data  = func() string { a, _ := filepath.Abs(filepath.Join(data, "prom2")); return a }()
+)
+
+func exec(cmd string, args ...string) error {
```
[the remark this answers is not in the document]
Comment (bwplotka) [reply examples/interactive/interactive_test.go:36]: it is used, just test is skipped ;p

File: examples/interactive/interactive_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (matej-g) [reply examples/interactive/interactive_test.go:43]: Yes, seems like skipping a test causes used funcs to be considered unused :shrug:

**Review by matej-g — APPROVED** [review]
Pending sorting out failed CI checks, looks good! :100:

**Comment by bwplotka** [comment]
Merging as it's extra example code. Fail is because of Katacoda being down so link check is  failing.

**Pull request merged**